axolotl-web/src/pages/MessageList.vue

```diff
@@ -437,13 +438,22 @@ export default {
     padding: 0px 20px;
 }
 
-.message{
-  padding:5px;
+.message {
+  padding: 5px 8px;
```

Comment:
This is not a color change but I felt like a bit more space on messages would make them easier to read.

Reply:
That's fine for me, i reduced paddings because someone mourned about wasting space...